modules/serverless-ossm-jwt-existing.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+// * serverless/networking/serverless-ossm-jwt.adoc
+
+[id="serverless-ossm-jwt-existing_{context}"]
+= Enable sidecar injection for an existing Knative service
+
+.Procedure
+
+. Add the `sidecar.istio.io/inject="true"` annotation to the Service resource
++
+
+.Example YAML
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: hello
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true" <1>
+    spec:
+      containers:
+      - image: docker.io/openshift/hello-openshift
+        name: container
+----
+<1> Add the `sidecar.istio.io/inject="true"` annotation.
+
+. Apply the Service resource YAML file:
++
+
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Create a policy that only allows requests with valid JSON Web Tokens (JWT) in your serverless application namespace:
+
+.. Copy the following into a YAML file:
++
+
+[source,yaml]
+----
+apiVersion: authentication.istio.io/v1alpha1
+kind: Policy
+metadata:
+  name: default
+spec:
+  origins:
+  - jwt:
+      issuer: testing@secure.istio.io
+      jwksUri: "https://raw.githubusercontent.com/istio/istio/release-1.6/security/tools/jwt/samples/jwks.json"
+      triggerRules:
+      - excludedPaths:
+        - prefix: /metrics
+        - prefix: /healthz
+  principalBinding: USE_ORIGIN
+----
+
++
+[IMPORTANT]
+====
+The paths `/metrics` and `/healthz` must be included in `excludedPaths`, as they are accessed from system pods in the `knative-serving` namespace.
+====
+
+.. Apply the YAML file:
++
+
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+.Verification steps
+
+. If you try to use a `curl` request, it is denied.
++
+.Example `curl` request
++
+
+[source,terminal]
+----
+$ curl http://hello-example-default.apps.knakayam-dev9.devcluster.openshift.com/
```

Comment:
Can you replace `hello-example-default.apps.knakayam-dev9.devcluster.openshift.com` with `hello-example-default.apps.mycluster.example.com`? 

`knakayam-dev9.devcluster.openshift.com` is my original cluster :smile: